### DISCUSSION TITLE
Icon Farbe - Welcher Eintrag bestimmt das beim Einschalten (true) vom DP das Icon die Farbe "gelb" bekommt

### DISCUSSION THREAD

**Discussion by mcuiobroker** [Zefau/ioBroker.jarvis#2720] [General]
Bislang dachte ich, wenn man das Gewerk "Licht" gewählt hat, dann werden bei den Geräte-DPs die Icon automatisch gelb, wenn sie eingeschaltet werden (true)
Jetzt stelle ich fest, dass egal was ich im Standard einstelle das Icon wird nur gelb, wenn ich es im Icon Stil einstelle.
Wo sieht man was bei welchen Gewerken die Standardeinstellung und die Auswirkungen sind (Farbeinstellungen)

**Comment by Zefau** [comment]
Icons sind immer primaryColor, wenn nichts durch den Benutzer definiert ist.
